dashboard/app/views/pd/application/teacher_application_mailer/accepted.html.haml

```diff
@@ -14,7 +14,13 @@
     in partnership with Code.org!
   - else
     = @application.course_name + "!"
-  Please read on for more details, and then make
+
+  - if @application.scholarship_status == Pd::ScholarshipInfoConstants::YES_CDO
```

Comment:
I wasn't 100% sure of this conditional -- let me know if it should be more permissive.

This was also a tough change to test as we don't have a preview for it. I tried to create one, but, because scholarship info is held in a different table, I didn't want to go too far down the rabbit hole of figuring out how to do that without saving things to the database (which we don't currently do for the previews). If we want a preview for this, I'd prefer to make another task. I did change the logic slightly to see this one sentence, so I know it looks correctly formatted:

<img width="1142" alt="Screenshot 2023-11-20 at 11 57 42 AM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/351bf5d6-397d-49f1-9619-cd14914c1ab8">